metadata/static/spark/overview.md

```diff
@@ -0,0 +1,26 @@
+## Overview
+Apache Spark is a fast and general-purpose cluster computing system for big data. 
+It provides high-level APIs in Scala, Java, Python, and R, and an optimized engine that supports general computation graphs for data analysis. 
+It also supports a rich set of higher-level tools including: Spark SQL for SQL and DataFrames, MLlib for machine learning, 
+GraphX for graph processing, and Spark Streaming for stream processing.
+
+The KUDO Spark Operator creates, configures, and manages instances of Spark Operator running on Kubernetes.
+
+- Easily create and manage Spark workloads on Kubernetes using `SparkApplication` resource
+- Spark History Server support
+- Metrics collection into Prometheus via ServiceMonitors  
+- Monitor Spark jobs using pre-configured Grafana dashboards
+- High Availability deployment mode
+- Integration with external batch scheduler (Volcano)
+- Graceful rolling updates for any cluster configuration changes
+- Graceful rolling upgrades when upgrading the operator version
```

Comment:
I'd remove that because of the nature of Spark Operator and Spark jobs being of finite duration as opposed to persistent data services. Another aspect is the incompatibility between major Spark versions and potential incompatibility between Spark Operator CRDs between upgrades.